package.json

```diff
@@ -9,7 +9,7 @@
     "build": "make all",
     "lint": "prettier --single-quote --semi false -l \"{src,test,examples}/**/*.js\" && eslint \"{src,test,examples}/**/*.js\"",
     "pretty": "prettier --single-quote --semi false --write \"{src,test,examples}/**/*.js\" && eslint --fix \"{src,test,examples}/**/*.js\"",
-    "test": "npm run lint && NODE_ENV=test jest --maxWorkers=2",
+    "test": "NODE_ENV=test jest --maxWorkers=2",
```

Comment:
dernit, didn't mean to commit this. the linter was blocking me from running the tests while i was figuring out how to get a generator to `yield` from within a `map` (you can't`) so i pulled it out. I'll throw it back in as I don't have enough context to suggest an alternative.